pages/index.vue

```diff
@@ -116,11 +107,11 @@
                     </div>
                 </div>
                 <div class="templates">
-                    <h2>Templates</h2>
-                    <p class="font-italic">Start by choosing a template</p>
+                    <h2>Layout</h2>
+                    <p class="font-italic">Do you need only a panel or a strip full of panels?</p>
 
                     <div class="comic-sections">
-                        <h3>Comic Panels</h3>
+                        <h3>Single Panel</h3>
```

Comment:
maybe keep Comic Panel - so we only use the word single for the Size of the Single Panel